tests/fetch_test.go

```diff
@@ -120,6 +132,8 @@ func TestFetchDoCommand(t *testing.T) {
 		video, ok := res["video"].(string)
 		test.That(t, ok, test.ShouldBeTrue)
 		test.That(t, video, test.ShouldNotBeEmpty)
+		filePath := pathForFetchCmd(fetchCmd1["from"].(string))
+		assertNoFile(t, filePath)
```

Comment:
Is there a point in these tests prior to this line that you should check that a file has been created?

Reply:
I would like to check for files created and deleted, but I would either need to refactor the concat flow to break the steps up for testing. Or, introduce some[ fs watcher ](https://github.com/fsnotify/fsnotify) to get events. I could create a follow up ticket for this.

Reply:
I think this is worth a follow-up ticket. Let's make sure code is testable from the get-go as much as we can!

Reply:
Sounds good. https://viam.atlassian.net/browse/RSDK-10259